libs/blocks/merch-card/merch-card.js

```diff
@@ -98,6 +99,12 @@ const parseContent = (el, merchCard) => {
     }
   });
 
+  const mnemonicList = el.querySelector('.mnemonic-list');
+  if (mnemonicList) {
+    bodySlot.append(mnemonicList);
+    await decorateMnemonicList(bodySlot);
```

Comment:
I wonder whether `bodySlot` already has some content here. If so, we should limit the scope of decoration to `mnemonicList`.

suggestion:
```
await decorateMnemonicList(mnemonicList);
bodySlot.append(mnemonicList);
```

Reply:
Updated. Mnemonics need to be inserted after paragraph content, so first decoration happens then insertion into body slot. LMK if this looks okay, otherwise I can make the foreach async and do all at once.